tendermint/src/rpc/client.rs

```diff
@@ -16,12 +16,13 @@ use hyper::header;
 pub struct Client {
     /// Address of the RPC server
     address: net::Address,
+    ws: Option<rpc::endpoint::subscribe::WebSocketEvents>,
```

Comment:
not sure but shouldn't this be a list of such events? See also the comment in `pub async fn subscribe` about this being overwritten for each `subscribe` call.